batterycelltesting/OCV TEST/OCV.py

```diff
@@ -0,0 +1,130 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mtick
+import os
+import argparse
+import zipfile 
+
+def weighting (current, valid_current):
+   low = min(valid_current)
+   high = max(valid_current)
+   if current >= 0:
+      weight = 1 - current / high #Weights added based on closeness to 0A
+   elif current <0:
+      weight = 1 - current / low
+   return weight
+
+def graphing (filename):
+   ax = plt.axes()
+   filename = filename[0:len(filename)-4]
+   plt.title(filename + " OCV Visualization")
+
+   plt.xlim(min(current),max(current))
+   plt.xlabel("Current(A)(X e10)")
+   plt.xticks(np.arange(min(current),max(current), 0.25))##spacing
```

Comment:
As well, what does the ## spacing comment mean? also python comments only need #